command/quota_apply.go

```diff
@@ -275,5 +277,21 @@ func parseQuotaResource(result *api.Resources, list *ast.ObjectList) error {
 		return err
 	}
 
+	// Find the network ObjectList, parse it
+	nw := listVal.Filter("network")
+	if len(nw.Items) < 1 {
+		return nil
```

Comment:
I'd avoid returning early here, as it complicates flow a little here and makes it a tad harder to add more parsing elements later.  I'd suggest either moving this check to `jobspec.ParseNetwork` where returning early is more ok, or have this block gated by `len(nw.Items) > 0` check